submissions/maryOkosun/phpSlim/app/controllers/ArticleController.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace App\Controller;
+
+use DB\DB;
+use PDO;
+use PDOException;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+
+class ArticleController
+{
+    private $db;
+
+    public function __construct()
+    {
+        $this->db = (new DB())->connect();
+    }
+
+    public function index(Request $request, Response $response, $args)
+    {
+        try {
+            $query = $this->db->prepare("select *
+            from articles
+            inner join admin
+            on articles.id = admin.id
+            where articles.created_by = admin.name;");
+            $query->execute();
+            $article = $query->fetchAll();
+            $response->getBody()->write(json_encode($article));
+            return $response->withHeader('Content-Type', 'application/json')->withStatus(200);
+        } catch (PDOException $ex) {
+            $response->getBody()->write(json_encode(["error" => $ex->getMessage()]));
+            return $response->withHeader('Content-Type', 'application/json')->withStatus(500);
+        }
+    }
+
+    public function getArticle(Request $request, Response $response, $args)
+    {
+        $id = $args['id'];
+        try {
+            $query = $this->db->prepare("select *
+            from articles
+            inner join admin
+            on articles.id = admin.id
+            where articles.created_by = admin.name and articles.id = :id;");
```

Comment:
This `articles.created_by = admin.name` shouldn't be part of your `WHERE` condition